### DISCUSSION TITLE
What is the provided map format?

### DISCUSSION THREAD

**Discussion by BielLopes** [larics/icuas22_competition#23] [Q&A]
@aivano. In the rulebook I have, the following text in exploration difficulty level 1: "Team will be provided with a map of the arena".
My question: what format will this map be provided? Where can I found a example of that map?
In another Q&A (https://github.com/larics/icuas22_competition/discussions/8#discussioncomment-2182587) have been said the obstacles have one single shape. Shoulded be very useful if the map to be simple as a list of the centroid of each pillar. Please consider providing a map in this format or having this information easily accessible, such as a grid of free and occupied cells.
Thanks in advance.

**Comment by fpetric** [comment]
Hi @BielLopes, 

thanks for asking! @aivano may correct me, but I am fairly sure that the map format that you will receive is an octomap, you can get the binvox.bt file for the current layout of the arena from [here](https://github.com/larics/larics_gazebo_worlds/tree/master/models/competition).
